examples/direct-upload/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Dwyl Direct Upload</title>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flat-ui/2.2.2/css/flat-ui.min.css">
+    <script type="text/javascript" charset="utf-8" src="./generate-keys.js"></script>
```

Comment:
are you `WebPack`-ing or `Browserify`-ing this code in order to have access to the `require`'d modules?